soh/src/code/sys_matrix.c

```diff
@@ -552,15 +552,15 @@ Mtx* Matrix_MtxFToMtx(MtxF* src, Mtx* dest) {
     return dest;
 }
 
-Mtx* Matrix_ToMtx(Mtx* dest, char* file, s32 line) {
-    FrameInterpolation_RecordMatrixToMtx(dest, file, line);
-    guMtxF2L(Matrix_CheckFloats(sCurrentMatrix, file, line), dest);
+Mtx* Matrix_ToMtx(Mtx* dest) {
+    FrameInterpolation_RecordMatrixToMtx(dest, __FILE__, __LINE__);
+    guMtxF2L(Matrix_CheckFloats(sCurrentMatrix), dest);
     return dest;
     //return Matrix_MtxFToMtx(Matrix_CheckFloats(sCurrentMatrix, file, line), dest);
 }
 
-Mtx* Matrix_NewMtx(GraphicsContext* gfxCtx, char* file, s32 line) {
-    return Matrix_ToMtx(Graph_Alloc(gfxCtx, sizeof(Mtx)), file, line);
+Mtx* Matrix_NewMtx(GraphicsContext* gfxCtx) {
+    return Matrix_ToMtx(Graph_Alloc(gfxCtx, sizeof(Mtx)));
```

Comment:
```suggestion
    return Matrix_ToMtx(Graph_Alloc(gfxCtx, sizeof(Mtx)), file, line);
```

Here's an actual example. This will preserve the line number in say actor code or engine code that actually called this chain of matrix functions. It will also keep the same semantics to what the original code did.